EIPS/eip-2381.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 2381
+title: ENS support for ERC 721 TokenID
+author: Oisin Kyne (@OisinKyne) <oisin@kyne.eu>
+discussions-to: https://github.com/ethereum/EIPs/issues/2381
+status: Draft
+type: Meta
+created: 2019-11-15
+requires (*optional): 137, 165, 721
+---
+
+<!--You can leave these HTML comments in your merged EIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new EIPs. Note that an EIP number will be assigned by an editor. When opening a pull request to submit your EIP, please use an abbreviated title in the filename, `eip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the EIP.-->
+
+_TL;DR_: This EIP makes ERC721 Non-Fungible Tokens addressable by the Ethereum Name Service by adding a new resolver profile, `uint256 tokenID()`.
+
+_Slightly longer TL;DR_: It makes sense to resolve an ENS name to a contract address for fungible tokens such as ERC20, as each token in the contract is indistinguishable from another. However, for non-fungible tokens, pointing to the contract address alone is not enough, as tokens within the contract are meant to be unique and distinguishable. Non-fungibles might have different valuations, might grant the holder different rights or rewards etc.
+
+This EIP proposes a new ENS resolver for ERC721's tokenID field. This allows the Ethereum Name Service to address a single non-fungible token within an ERC721 token contract of many tokens.
```

Comment:
Simple Summary should just be the TL;DR (but without that prefix).  It is meant to be a one-line description of what this standard is about.  The abstract is where you would put a more verbose description, usually a human-readable and terse version of the specification.

Also, EIPs should be worded as though read by a future reader.  This means that EIPs do not propose, they assert.  "This EIP describes a new ENS resolver for ERC721's tokenID field".

Reply:
TL;DRs removed and EIP reworded to sound post-fact rather than like a proposal.